docs/tx_rollups.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Transaction Optimistic Rollups
+id: tx_rollups
+author: Davis Sawali
+---
+
+A Transaction Optimistic Rollup (TORU) is an experimental temporary scaling solution to help Tezos developers and users get acquainted with the idea of working with rollups. 
+
+This is a trimmed, more concise documentation of how to do TORU operations in Taquito. If you aren't yet familiar with rollup node interactions, please refer to this [documentation](https://tezos.gitlab.io/alpha/transaction_rollups.html?highlight=transaction%20rollups#transaction-rollups) by Nomadic Labs.
+
+Some new terms and jargon is introduced in the Jakartanet update regarding rollup nodes, please refer to this [glossary](https://)
```

Comment:
The link to the glossary is missing

Reply:
Ah good catch, that was just a placeholder. Do you think we should create one ourselves on the bottom of the page?